ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -158,6 +163,24 @@ public class HelixBootstrapUpgradeUtil {
   private static final Logger logger = LoggerFactory.getLogger("Helix bootstrap tool");
   private static final String ALL = "all";
 
+  // Waged auto rebalancer default configs. TODO: Provide them as configurable inputs to tool.
+  static final String DISK_KEY = "DISK";
+  static final String RACK_KEY = "rack";
+  static final String HOST_KEY = "host";
+  static final String TOPOLOGY = "/" + RACK_KEY + "/" + HOST_KEY;
+  static final String FAULT_ZONE_TYPE = "rack";
+  static final int MAX_PARTITIONS_IN_TRANSITION = 50;
+  static final int DELAY_REBALANCE_TIME_IN_MS = 1800000;
+  static final int LESS_MOVEMENT = 2;
+  static final int EVENNESS = 3;
+  // the default of partition. each partition can store up to 24 * 16 = 384 GB data, and leave 2 GB here for margin
+  static final int DEFAULT_PARTITION_WEIGHT = 100;
+  // Allow a host to be filled upto 85%
+  static final int INSTANCE_MAX_CAPACITY_PERCENTAGE = 85;
```

Comment:
it should be a bit higher, helix should be able to leverage more than 85% of the host capacity. It should be at least 95. This is the limit that helix can use. When we switch to FULL_AUTO, the current host usage should be roughly 85%.